Hymns/Common/SoundCloudWebView.swift

```diff
@@ -0,0 +1,66 @@
+import SwiftUI
+import WebKit
+
+struct SoundCloudWebView: UIViewRepresentable {
```

Comment:
This doesn't build for me because SoundCloudWebView is not added in the project.pbxproj

Reply:
That's wierd....

Reply:
Should be fixed now. I recommitted, that was super wierd.